cs-http-client/src/main/java/io/cloudslang/content/httpclient/build/conn/SSLConnectionSocketFactoryBuilder.java

```diff
@@ -68,25 +68,26 @@ public SSLConnectionSocketFactory build() {
         boolean trustAllRoots = Boolean.parseBoolean(trustAllRootsStr);
 
         SSLContextBuilder sslContextBuilder = SSLContexts.custom();
+        String changeit = "changeit";
+        String javaKeystore = System.getProperty("java.home") + "/lib/security/cacerts";
         if (!trustAllRoots) {
-            boolean useClientCert = !StringUtils.isEmpty(keystore);
+            boolean useClientCert = StringUtils.isNotEmpty(keystore);
```

Comment:
@mihaitusa  It is fine like this, we don't want to introduce a change in behaviour right now, even though a slight small one